### PR TITLE
Closes #9 - Display Default page

### LINKED ISSUE #9: Display Default Page
Update reminders to have a conditional that will only display notes if notes exist.

There should be a default welcome page welcoming users to the app and/or directions to create a new note. Feel free to be creative here.

### PR DESCRIPTION
**Pull request by MilkMan90**
@martensonbj @Tman22 @nfosterky @stevekinney 

## Purpose

Update reminders to have a conditional that will only display notes if notes exist.

There should be a default welcome page welcoming users to the app and/or directions to create a new note. Feel free to be creative here.

## Approach

It solves it by adding a conditional to check if reminders exist or not

### Learning

[EmberJS Conditionals](https://guides.emberjs.com/v2.5.0/templates/conditionals/)

### Test coverage 

We added 1 test to make sure that when we load a page with no conditionals, the "welcome to your notes" paragraph shows up.

### Follow-up tasks

- #10 

### Screenshots

screen shot 2016-12-08 at 3 56 51 pm

### CODE REVIEW

File: app/templates/reminders.hbs [hunk 1]
```diff
@@ -7,6 +8,10 @@
     {{/link-to}}
   {{/each}}
 </ul>
+{{else}}
```
Comment (nfosterky) [inline app/templates/reminders.hbs]: Make sure to indent the content of your if/else block

**Comment by MilkMan90** [comment]
@nfosterky fixed =)

File: app/templates/reminders.hbs [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+  <ul class="reminder-list">
+    {{#each model as |reminder|}}
+      {{#link-to 'reminders.reminder' reminder}}
+        <li class='spec-reminder-item'>
+            {{reminder.title}}
+        </li>
+      {{/link-to}}
+    {{/each}}
+  </ul>
+  {{else}}
```
Comment (nfosterky) [inline app/templates/reminders.hbs]: The `{{#else}}` should be the same indentation as the `{{#if model}}` block

**Comment by MilkMan90** [comment]
@nfosterky ok it's actually fixed this time.

**Pull request merged**